qa/src/test/java/org/eclipse/kapua/service/StepData.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
```

Comment:
So, as I understand it would be enough to just state the year when the file was introduced.
But it looks like in our case there is a year when file was created and the second year is year of change.